packages/x-charts/src/ScatterChart/ScatterMarker.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { SeriesId } from '../models/seriesType/common';
+import { useScatterSeries } from '../hooks';
+
+export interface ScatterMarkerProps {
+  ownerState: ScatterMarkerOwnerState;
+  /**
+   * Callback fired when clicking on a scatter item.
+   * @param {MouseEvent} event Mouse event recorded on the `<svg/>` element.
+   */
+  onClick?: (event: React.MouseEvent<SVGElement, MouseEvent>) => void;
+}
+
+export interface ScatterMarkerOwnerState {
+  /**
+   * ID of the series this marker belongs to.
+   */
+  id: SeriesId;
+  dataIndex: number;
+  x: number;
+  y: number;
+  color: string;
+  isHighlighted: boolean;
+  isFaded: boolean;
+}
+
+/**
+ * TODO: Document
+ */
+function ScatterMarker({ ownerState, ...other }: ScatterMarkerProps) {
+  const series = useScatterSeries(ownerState.id);
```

Comment:
This `useScatterSeries` seems unnecessary, the prop can be passed in for performance purposes.

Reply:
Makes sense 👍 I'll do that once it's clear whether to use `ownerState` or not